server/pairing/sync_device_test.go

```diff
@@ -363,13 +363,9 @@ func defaultSettings(generatedAccountInfo generator.GeneratedAccountInfo, derive
 	visibleTokenJSONRaw := json.RawMessage(visibleTokensJSON)
 	settings.WalletVisibleTokens = &visibleTokenJSONRaw
 
-	networks := make([]map[string]string, 0)
-	networksJSON, err := json.Marshal(networks)
-	if err != nil {
-		return nil, err
-	}
-	networkRawMessage := json.RawMessage(networksJSON)
-	settings.Networks = &networkRawMessage
+	networks := `[{"id":"goerli_rpc","chain-explorer-link":"https://goerli.etherscan.io/address/","name":"Goerli with upstream RPC","config":{"NetworkId":5,"DataDir":"/ethereum/goerli_rpc","UpstreamConfig":{"Enabled":true,"URL":"https://goerli-archival.gateway.pokt.network/v1/lb/3ef2018191814b7e1009b8d9"}}},{"id":"mainnet_rpc","chain-explorer-link":"https://etherscan.io/address/","name":"Mainnet with upstream RPC","config":{"NetworkId":1,"DataDir":"/ethereum/mainnet_rpc","UpstreamConfig":{"Enabled":true,"URL":"https://eth-archival.gateway.pokt.network/v1/lb/3ef2018191814b7e1009b8d9"}}}]`
+	var networksRawMessage json.RawMessage = []byte(networks)
+	settings.Networks = &networksRawMessage
```

Comment:
We will need to ensure that `NodeConfig.RootDataDir` is always present in the pairing context.

```go
type NodeConfig struct {
	// NetworkID sets network to use for selecting peers to connect to
	NetworkID uint64 `json:"NetworkId" validate:"required"`

	RootDataDir string `json:",omitempty"`

	// DataDir is the file system folder the node should use for any data storage needs.
	DataDir string `validate:"required"`

```

Reply:
I feel not well 🥹 because i saw this alert with #3303 before: "⚠️ NodeConfig ⚠️" , It is obvious that it did not occupy my heart.

Reply:
@Samyoul fixed